### ISSUE TITLE
mat64: TestDet is flakey

### ISSUE THREAD

**Issue by kortschak** [gonum/matrix#437]
With the addition of a new independent test, TestDet now fails with

```
--- FAIL: TestDet (0.00s)

	list_test.go:605: Answer mismatch: DetVsChol(*mat64.basicSymmetric), size: struct { ar int; ac int }{ar:6, ac:6}, atrans true
```

I imagine this is due to an unfortunate random value stream position. Changing the tolerance to 1e-11 allows the test to pass, but it is curious that it only fails with `*mat64.basicSymmetric` and not `*mat64.SymDense` (though maybe the bad stream is very rare), so I don't want to just rush in and broaden the tolerance for this test.

/cc @btracey

**Comment by kortschak** [comment]
`go test -run TestDet -count 1000 | grep -c FAIL`  gives 82 in a variety of types. So it's just rare.

**Comment by kortschak** [comment]
Closed by #438.
